tokio-trace/README.md

```diff
@@ -14,10 +14,10 @@ often be quite challenging. Since individual tasks are multiplexed on the same
 thread, associated events and log lines are intermixed making it difficult to
 trace the logic flow. `tokio-trace` expands upon logging-style diagnostics by
 allowing libraries and applications to record structured events with additional
-information about *temporality* and *causality* — unlike a log message, a span
+information about _temporality_ and _causality_ — unlike a log message, a span
```

Comment:
This looks like an unrelated change?

Reply:
ah, that's my editor's autoformat, whoops